sdk-setup/src/mb2

```diff
@@ -2763,8 +2791,9 @@ run_deploy() {
     local device_rpms=$(sed "s,^$OPT_OUTPUTDIR/,RPMS/," <<<"$rpms")
 
     if [[ $dry_run_opt ]]; then
-        echo "The following packages would be deployed:"
-        sort <<<"$rpms"
+        echo "The following packages would be considered for deployment:"
+        sort <<<"$rpms" |sed 's/^/  /'
+        echo
```

Comment:
Instead of just listing packages to be considered, perhaps we should actually run zypper dup --dry-run? That would list the packages which will really be deployed, wouldn't it?

Reply:
And of course you did it in the next commit :-D